frontend/src/app/register/register.ctrl.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Controller for register page
+ */
+angular.module('flocs.user')
+.controller('registerCtrl', ['$scope','$location','$log','userDao', 
+    function($scope,$location,$log,userDao){
+
+	    function send(){
```

Comment:
Function is supposed to send information from formular to service. Is it
really intended to name this function to register?

On Mon, Nov 23, 2015 at 3:11 PM, Jiří Mauritz <notifications@github.com>
wrote:

> In frontend/src/app/register/register.ctrl.js
> <https://github.com/effa/flocs/pull/56#discussion_r45606178>:
>
> > @@ -0,0 +1,22 @@
> > +/*
> > + * Controller for register page
> > + */
> > +angular.module('flocs.user')
> > +.controller('registerCtrl', ['$scope','$location','$log','userDao',
> > +    function($scope,$location,$log,userDao){
> > +
> > +	    function send(){
>
> rename to register()
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/effa/flocs/pull/56/files#r45606178>.
>



-- 
Best regards,

Jakub Peschel